cms/static/js/views/paging_header.js

```diff
@@ -78,11 +83,11 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
             totalItemsCountLabel: function() {
                 var totalItemsLabel;
                 // Translators: turns into "25 total" to be used in other sentences, e.g. "Showing 0-9 out of 25 total".
-                totalItemsLabel = interpolate(gettext('%(total_items)s total'), {
-                    total_items: this.view.collection.totalCount
+                totalItemsLabel = interpolate(gettext('%(totalItems)s total'), {
+                    totalItems: this.view.collection.getTotalRecords()
                 }, true);
-                return interpolate('<span class="count-total">%(total_items_label)s</span>', {
-                    total_items_label: totalItemsLabel
+                return interpolate('<span class="count-total">%(totalItemsLabel)s</span>', {
```

Comment:
Convert this to use the safe template best practices: http://ui-toolkit.edx.org/utilities/html-utils/